rpcs3/main.cpp

```diff
@@ -508,6 +508,19 @@ int main(int argc, char** argv)
 	setenv( "KDE_DEBUG", "1", 0 );
 #endif
 
+#ifdef __APPLE__
+	struct ::rlimit rlim;
+	::getrlimit(RLIMIT_NOFILE, &rlim);
+	rlim.rlim_cur = OPEN_MAX;
+	if (::setrlimit(RLIMIT_NOFILE, &rlim) != 0)
+		std::cerr << "Failed to set max open file limit (" << OPEN_MAX << ").\n";
```

Comment:
why is this cerr

Reply:
I copied that from the Linux code.

Reply:
This seems to have fixed a couple games with the Open File limit error. Notably `Tony Hawk's® Pro Skater™ 5 [BLUS31568]` Keep up the great work Nastys!!!!

Reply:
It was likely done like that for linux since the app will be opened via the commandline, so stderr is highly visible. Not sure that makes sense on macos.

Reply:
You can open it on macOS in a terminal window, and you don't necessarily have to open it in a terminal window on Linux.

Reply:
The idea is that we can see this in the log, instead of just relying on insufficient user statements.
You can do both cerr and log error for both occasions.

Reply:
Line 521 should log maxfiles to file on both macOS and Linux.